pkg/kubectl/cmd/cp/cp.go

```diff
@@ -118,28 +129,43 @@ var (
 )
 
 func extractFileSpec(arg string) (fileSpec, error) {
-	if i := strings.Index(arg, ":"); i == -1 {
+	if len(arg) == 0 {
+		return fileSpec{}, errFileCannotBeEmpty
+	}
+	switch arg[0] {
+	case '.', '/', ':':
+		return fileSpec{File: arg}, nil
+	}
+
+	i := strings.Index(arg, ":")
+	if i == -1 {
```

Comment:
`i < 0` is safer check, just in case in the futre `strings.Index` will return different error codes.

Reply:
Ahh, sure.

If they decide to switch one day, I don't wan't to think about all the things that will break :sweat_smile: